client/core/core.go

```diff
@@ -6686,6 +6674,48 @@ func (c *Core) initialize() error {
 	return nil
 }
 
+// connectAccount makes a connection to the DEX for the given account. If a
+// non-nil dexConnection is returned, it was inserted into the conns map even if
+// the initial connection attempt failed (connected == false), and the connect
+// retry / keepalive loop is active.
+func (c *Core) connectAccount(acct *db.AccountInfo) (dc *dexConnection, connected bool) {
```

Comment:
Can remove this method.  We don't use it anymore.  It was only designed for `initialized`, and is simpler doing just what's needed there.

Reply:
Ohh, you had this PR up when `initialize` was using `connectAccount`... which would have allowed the connect loop to run.  Sorry. You can either wire `connectAccount` back into `initialize` or change `initialize` to do it.

Reply:
Okay.